shared/chat/conversation/info-panel/add-people.js

```diff
@@ -0,0 +1,77 @@
+// @flow
+import * as React from 'react'
+import {Box2, Button, FloatingMenu, OverlayParentHOC, type OverlayParentProps} from '../../../common-adapters'
+import {connect} from '../../../util/container'
+import * as RouteTreeGen from '../../../actions/route-tree-gen'
+import {teamsTab} from '../../../constants/tabs'
+
+type Props = {
+  attachTo: () => ?React.Component<any>,
+  visible: boolean,
+  onAddPeople: () => void,
+  onHidden: () => void,
+}
+
+const AddPeopleHow = (props: Props) => {
+  const items = [{onClick: props.onAddPeople, title: 'To team'}, {disabled: true, title: 'To channel'}]
```

Comment:
No 'To channel' option for small teams, also the spacing seems off in the small team info pane:

<img width="319" alt="image" src="https://user-images.githubusercontent.com/11968340/52813622-27902180-3068-11e9-84b5-9da98692fd0c.png">